### PR TITLE
[PBW-3836] Add iframe.html to dynamically load a player and asset.

### PR DESCRIPTION
**Pull request by aldigun**

### CODE REVIEW

**Comment by aldigun** [comment]
Need to be at same time as https://github.com/ooyala/skin-config/pull/95

File: js/components/sharePanel.js [hunk 1]
```diff
@@ -66,11 +66,15 @@ var SharePanel = React.createClass({
     }
 
     else if (this.state.activeTab === this.tabs.EMBED) {
+      var iframeURL = this.props.skinConfig.shareScreen.embed.source
```
Comment (aldigun) [inline js/components/sharePanel.js]: Need a safety to non existing embed.source

**Comment by mistermendez** [comment]
@aldigun conflicts

File: iframe.html [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+  <script src="//player.ooyala.com/static/v4/candidate/latest/skin-plugin/html5-skin.js"></script>
+  <link rel="stylesheet" href="//player.ooyala.com/static/v4/candidate/latest/skin-plugin/html5-skin.css"/>
+</head>
+
+<body style="padding:0; margin:0; width:100%; height:100%; overflow:hidden">
+  <div id='container' style='width:100%; height:100%;'></div>
+  <script>
+    var queryParams = window.location.href;
+    var playerParam = {
+      "pcode": queryParams.match(/pcode=([^&]*)/)[1],
```
Comment (mistermendez) [inline iframe.html]: Uncaught TypeError: Cannot read property '1' of null

File: iframe.html [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+      "playerBrandingId": queryParams.match(/pbid=([^&]*)/)[1],
```
Comment (mistermendez) [inline iframe.html]: Uncaught TypeError: Cannot read property '1' of null

**Comment by mistermendez** [comment]
iframe works with updates in corresponding skin config - https://github.com/ooyala/skin-config/pull/95

and locally with valid query string params - http://0.0.0.0:8000/build/iframe.html?ec=RmZW4zcDo6KqkTIhn1LnowEZyUYn5Tb2&pbid=26e2e3c1049c4e70ae08a242638b5c40&pcode=5zb2wxOlZcNCe_HVT3a6cawW298X

:+1:

**Pull request merged**